src/java/main/org/apache/zookeeper/server/admin/Commands.java

```diff
@@ -329,12 +329,20 @@ public CommandResponse run(ZooKeeperServer zkServer, Map<String, String> kwargs)
             response.put("open_file_descriptor_count", osMbean.getOpenFileDescriptorCount());
             response.put("max_file_descriptor_count", osMbean.getMaxFileDescriptorCount());
 
+            response.put("last_client_response_size", stats.getClientResponseStats().getLast());
```

Comment:
Please update zookeeperAdmin.xml about these commands. Please refer monitoring the health of the cluster section in this page.

Reply:
Also, I could see contrib/monitoring/ganglia/zookeeper_ganglia.py file contains metrics section. Probably you could refer 'zk_open_file_descriptor_count': {'units': 'descriptors'}, as an example case.